lldb/source/Plugins/Language/Swift/LanguageSwiftProperties.td

```diff
@@ -0,0 +1,12 @@
+include "../../../../include/lldb/Core/PropertiesBase.td"
```

Comment:
I think clang-format ruined this file. I would restore the fail to its original form, apply your changes, and make sure `git clang-format` doesn't touch this file. We should probably find a way to exclude these tablegen files from getting picked up by `git clang-format`

Reply:
We could either add a `// clang-format off` and `// clang-format on` at the top and bottom of the file, or add a `.clang-format-ignore` file along `llvm-project/lldb/.clang-format`. I think the latter is the better option as we will probably need to ignore other files in the future.